src/app/assets/languages/en.json

```diff
@@ -166,7 +166,7 @@
       "add-entry-form-value-tooltip": "The value can contain simple values ...",
```

Comment:
@karesti Hi, I've noticed the tooltips for keys and values for the create entry dialog are incomplete. 

"The key can contain simple values ..." and "The value can contain simple values ..." 

How should we describe this? The values for keys and values depends on the encoding, right?

Reply:
yes, we can add json or protobuf entities, depending on the encoding of the cache

Reply:
The key can contain JSON or Protobuf values, depending on cache encoding.